SU-HSDO/suhumsci
# feat(STN-525) Update lockup styles

# READY FOR REVIEW

## Summary
This is a refinement of the lockup styles to reflect new branding guidelines for the default lockup.
These branding guidelines are being injected on live sites via the Asset Injector styles.

You can see the asset injector styles in the JIRA card or [in your local site](http://sparkbox-sandbox.suhumsci.loc/admin/config/development/asset-injector/css/lockup).

This work puts this styling into the themes for permanent use.

JIRA Card: https://sparkbox.atlassian.net/browse/STN-525

## Need Review By (Date)
1/13

## Urgency
medium

## Steps to Test
1. npm run test
2. [Disable the Asset Injector sheet on local](http://sparkbox-sandbox.suhumsci.loc/admin/config/development/asset-injector/css/lockup).
2.  Check the lockup with the default masthead and make sure it follows the correct styling.
- Check against these sites that are using the Asset Injector that are currently live:
- Colorful https://creees.stanford.edu/, https://economics.stanford.edu/, https://mediterraneanstudies.stanford.edu/,
- Traditional https://politicalscience.stanford.edu/, https://amstudies-prod.stanford.edu/, https://buddhiststudies.stanford.edu/
3. Verify that these styles should not affect the Legacy sites.
4. Add text to each line combination in the lockup to verify all the borders and text styles have been updated.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
